methods.ts

```diff
@@ -1228,7 +1228,7 @@ export type ApiMethods<F> = {
     business_connection_id: string;
   }): BusinessConnection;
 
-  /** Use this method to change the list of the bot's commands. See https://core.telegram.org/bots#commands for more details about bot commands. Returns True on success. */
+  /** Use this method to change the list of the bot's commands. @See https://core.telegram.org/bots/features#commands for more details about bot commands. Returns True on success. */
```

Comment:
```suggestion
  /** Use this method to change the list of the bot's commands. See https://core.telegram.org/bots/features#commands for more details about bot commands. Returns True on success. */
```
The docs string is actually copied from the website, so it's not supposed to be a JSDoc annotation